lib/hero-image-header/index.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component, PropTypes } from 'react';
+var Scroll = require('react-scroll');
```

Comment:
I would *suggest* adding some sort of detail in an issue or readme as to why we chose to use https://github.com/fisshy/react-scroll (*despite it's lack of tests...*), are we relying on the fact that it has a few thousand users to file issues and debug it if there is a browser incompatibility...? It works *now*...